### ISSUE TITLE
Mongo db and homestead

### ISSUE THREAD

**Issue by stojankukrika** [laravel/homestead#839]
Hi,
Did you plan to add for example MongoDB in this awesome package?

**Comment by stojankukrika** [comment]
Tnx. Sepcify like mariadb in fop of file? I didn't find it in manual. Can I specify how to naming databases,like mariadb?

**Comment by mikeaag** [comment]
Yeah, at the top of the file (though i'd guess it could probably go anywhere in the file).

Databases are defined in the same way under the `databases:` section

**Comment by stojankukrika** [comment]
What to do if I need mongodb and mariadb? For example, I need some of the databases to be mongodb and some mariadb. How to define them which type will be?

**Comment by stojankukrika** [comment]
TNX a loot @svpernova09 for info and avatar :)
